packages/clay-date-picker/src/index.tsx

```diff
@@ -345,105 +346,110 @@ const ClayDatePicker: React.FunctionComponent<IProps> = React.forwardRef<
 		const handleCalendarButtonClicked = () => setExpanded(!expanded);
 
 		return (
-			<div className="date-picker">
-				<ClayInput.Group id={id} ref={triggerElementRef}>
-					<ClayInput.GroupItem>
-						<InputDate
-							{...otherProps}
-							ariaLabel={ariaLabels.input}
-							currentTime={currentTime}
-							dateFormat={dateFormat}
-							disabled={disabled}
-							inputName={inputName}
-							onChange={inputChange}
-							placeholder={placeholder}
-							ref={ref}
-							time={time}
-							useNative={useNative}
-							value={value}
-						/>
-						{!useNative && (
-							<ClayInput.GroupInsetItem after>
-								<Button
-									className="date-picker-dropdown-toggle"
-									data-testid="date-button"
-									disabled={disabled}
-									displayType="unstyled"
-									onClick={handleCalendarButtonClicked}
-								>
-									<Icon
-										spritemap={spritemap}
-										symbol="calendar"
-									/>
-								</Button>
-							</ClayInput.GroupInsetItem>
-						)}
-					</ClayInput.GroupItem>
-				</ClayInput.Group>
-
-				{!useNative && (
-					<DropDown.Menu
-						active={expanded}
-						alignElementRef={triggerElementRef}
-						className="date-picker-dropdown-menu"
-						data-testid="dropdown"
-						onSetActive={setExpanded}
-						ref={dropdownContainerRef}
-					>
-						<div className="date-picker-calendar">
-							<DateNavigation
-								ariaLabels={ariaLabels}
-								currentMonth={currentMonth}
+			<FocusScope arrowKeysLeftRight>
```

Comment:
I think that here, it doesn't make much sense to enable the `arrowKeysLeftRight`, the behavior is very strange. I think keeping the focus is enough to use Tab and Shift+Tab.

I think the use of arrow keys here must be its own implementation because the behavior can be different, but we have to evaluate with Lexicon, as this is based on the native... (And it looks like the new native Date Picker looks very different now, you get stuck in a loop with navigation 🤷‍♂️)

Reply:
I was wondering navigating using left and right navigation due to https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html, However I noticed We aren't reaching the same result of this example w3c example.

Hey @draconux, Do you have a design/a11y perspective on this?

Reply:
I've created [LEXI-975](https://issues.liferay.com/browse/LEXI-975) to work on a new definition but there is no priority to do it (as far as I know). 

About the focus management, I think we should follow the W3 approach. 

- Thus, TAB and Shift + TAB to move the focus between focusable elements (orange numbers).
- Once the focus enters in the date zone (Red):
- If the user press TAB again the focus moves to the next area (Blue)
- ↑↓ are enabled to move the focus within the blue area

![focus-date-picker](https://user-images.githubusercontent.com/21216422/86333561-5d797a00-bc4c-11ea-911d-390071bc2ca1.png)

But I need time to work on it. My advice is that we are following this approach as we defined in other components such as the Vertical Bar Tabs.

Reply:
Thanks @drakonux for the information. @diegonvs I think we can deal with that later, we disable `arrowKeysLeftRight` for DatePicker. And we hope for the implementation so that @drakonux has more time to think about it.